rileylink/src/main/java/info/nightscout/androidaps/plugins/pump/common/hw/rileylink/ble/RileyLinkBLE.java

```diff
@@ -58,13 +60,11 @@
     private Runnable radioResponseCountNotified;
     private boolean mIsConnected = false;
 
-
+    @Inject
     public RileyLinkBLE(HasAndroidInjector injector, final Context context) {
-        injector.androidInjector().inject(this);
```

Comment:
Seems like we don't need `injector` here anymore

Reply:
oops, that slipped through. That would probably even crash the app because of a double injection

Reply:
Oh, I misread the comment, you're talking about the injector itself. I'd keep it just in case for now, it doesn't really hurt.